examples/http-ts/src/controller.ts

```diff
@@ -0,0 +1,35 @@
+import { IncomingMessage, ServerResponse } from 'http';
+import * as store from '@leapfrogtechnology/async-store';
+
+import { sayHi } from './service';
+import * as logger from './logger';
+
+/**
+ * Controller for `/home` endpoint.
+ *
+ * @param {IncomingMessage} req
+ * @param {ServerResponse} res
+ */
+export function home(req: IncomingMessage, res: ServerResponse) {
+  logger.info('Home controller called.');
+
+  res.write('<h1>Home</h1>');
+  res.write(`<p>${sayHi()}</p>`);
+  res.end();
+}
+
+/**
+ * Controller for `/other` endpoint.
+ *
+ * @param {IncomingMessage} req
+ * @param {ServerResponse} res
+ */
+export function other(req: IncomingMessage, res: ServerResponse) {
```

Comment:
No need.

Reply:
Okay

Reply:
Should I remove the `routes` as well and always call the `home` controller for all request???

Reply:
Yes. Try to simplify and minimize the example as much as possible but needs to demonstrate the use of async store in the fullest though.